log/init.go

```diff
@@ -17,12 +17,12 @@ func init() {
 }
 
 func initialize() {
-	value, err := snapctl.Get("debug").Run()
+	value, err := exec.Command("snapctl", "get", "debug").CombinedOutput()
```

Comment:
Executing command instead of using snapctl package to avoid circular dependency imports.

Reply:
Are package `snapctl` and `log` imported from each other?

Reply:
I have made this particular change so that they don't import each other.